alfalfa_worker/lib/alfalfa_connections_base.py

```diff
@@ -12,7 +12,7 @@
 from influxdb import InfluxDBClient
 
 
-class AlfalfaConnections:
+class AlfalfaConnectionsBase(object):
```

Comment:
need to inherit from `object` in python2 for base classes. I look forward to moving this all to python 3.